README.adoc

```diff
@@ -86,27 +57,27 @@ include::{common-includes}/gitclone.adoc[]
 
 === Try what you'll build
 
-The `finish` directory in the root of this guide contains the finished fault tolerance implementation
+The `finish` directory in the root of this guide contains the finished implementation
 for the application. You can give it a try before you proceed with building your own.
 
-To try out the application, first navigate to the `finish` directory. Then, execute the following
-Maven goals to build the application and run it inside Open Liberty:
+To try out the application, navigate to the `finish` directory and run the following command:
 
 ```
-cd finish
 mvn clean install liberty:start-server
 
 ```
 
-Point your browser to the `http://localhost:9080/inventory/systems/localhost` URL. You see a result
-in JSON format with the system properties of your local JVM. When you visit this URL, some of these system
+Point your browser to the `http://localhost:9080/inventory/systems/localhost` URL, which accesses the
+`inventory` service with a `localhost` host name. You see the system properties for this host.
+When you visit this URL, some of these system
 properties, such as the OS name and user name are automatically stored in the inventory.
 
 Now, open the `resources/CustomConfigSource.json` file and change the
 `io_openliberty_guides_system_inMaintenance` property from `false` to `true` and save the file. You do not need
 to restart the server. Next, return to your browser and point back to the
 `http://localhost:9080/inventory/systems/localhost` URL.
-Because the `system` service is now in maintenance, you see the cached properties for this localhost.
+The fallback mechanism is triggered because the `system` service is now in maintenance.
+You see the cached properties for this `localhost`.
```

Comment:
Because we're referring to the "localhost" as such, we don't need the code phrase format.

Reply:
Fixed